meta/SearchConfig.php

```diff
@@ -36,8 +36,10 @@ class SearchConfig extends Search
     /**
      * SearchConfig constructor.
      * @param array $config The parsed configuration for this search
+     * @param bool $renderMetadata If this is true, then dynamic filters will not be parsed.
+     *                             This is for metadata rendering.
      */
-    public function __construct($config)
+    public function __construct($config, $renderMetadata=false)
```

Comment:
Minor thing: I would rename the parameter to be independent from it's current usage and align it with what it actually does. Eg. `$applyDynamicFilters` or something similar

Reply:
Code of __construct not only blocks dynamic filter, but also blocks limit parameter already defined with aggregation syntax. Is ``$applyDynamicFilters'' still valid in this case?